packages/web/src/store/application/ui/editFolderModal/slice.ts

```diff
@@ -0,0 +1,32 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+
+export type EditFolderModalState = {
+  isOpen: boolean
+  folderId: string | null
+}
+
+const initialState: EditFolderModalState = {
+  isOpen: false,
+  folderId: null
+}
+
+type OpenPayload = string
+
+const slice = createSlice({
+  name: 'application/ui/editFolderModal',
+  initialState,
+  reducers: {
+    open: (state, action: PayloadAction<OpenPayload>) => {
+      state.isOpen = true
```

Comment:
so it's a bit confusing, but i think we are trying to handle modal open/closed using the ui/modal slice. so this slice would just care about the data the modal needs, and then opening/closing is managed by the ui/modal slice. share-modal/slice has an example of how that is done!